src/common/RootProviders.tsx

```diff
@@ -29,12 +30,14 @@ export function RootProviders({
             <OAuthContextProvider authConfig={authConfig}>
               <ActiveAccountContextProvider>
                 <ActiveProjectContextProvider>
-                  <NoInternetToastProvider>
-                    <SafeAreaProvider>
-                      <NavigationContainer>{children}</NavigationContainer>
-                      <Toast />
-                    </SafeAreaProvider>
-                  </NoInternetToastProvider>
+                  <BrandConfigProvider>
```

Comment:
The Paper Provider which is wrapped in the BrandConfig provider needs to be outside any UI elements, but inside most other networking or store providers because `Modal` components are portaled here, at the top level.